B/Bun/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Bun"
+version = v"1.2.10"
+
+release_url = "https://github.com/oven-sh/bun/releases/download/bun-v$version"
+
+# We download the SHA file to extract the SHAs of the archives, taken from https://github.com/MichaelHatherly/BundledWebResources.jl
+sha256sums = let 
+    d = Dict{String,String}()
+    sha256sums_url = "$(release_url)/SHASUMS256.txt"
+    for line in eachline(download(sha256sums_url))
+        sha, file = strip.(split(line, ' '; limit = 2))
+        d[file] = sha
+    end
+    d
+end
+
+repo_SHA1 = let
+    response = read(download("https://api.github.com/repos/oven-sh/bun/git/refs/tags/bun-v$(version)"), String)
+    match(r"\"sha\": \"([0-9a-f]{40})\"", response).captures[1]
+end
+
+# This will store filename, arch and platform for each supported platform
+data = [
+    (; filename = "bun-linux-x64.zip", sha = sha256sums["bun-linux-x64.zip"], target = "x86_64-linux-gnu", platform = Platform("x86_64", "linux"; libc = "glibc")),
```

Comment:
Any reason for leaving out https://github.com/oven-sh/bun/releases/download/bun-v1.2.10/bun-linux-x64-musl.zip ?

Reply:
not really, didn't realize this was available. Updated the recipe to add this